doc/how_to/editor/pycharm_configure.md

```diff
@@ -40,3 +40,13 @@ With the configuration in place, you can now easily `run` Panel apps via the `pa
 With the configuration in place, you can now easily `debug` Panel apps via the `panel serve` configuration. The picture below shows debugging in action, where a breakpoint is hit when the Button is clicked.
 
 ![Debug Panel App in PyCharm](../../_static/images/pycharm-debug-panel.png)
+
+## Notebook Environment
+
+PyCharm Professional offers support for [Jupyter Notebooks](https://www.jetbrains.com/help/pycharm/ipython-notebook-support.html) and [ipywidgets](https://www.jetbrains.com/help/pycharm/interactive-js-widgets.html), enhancing interactive Python development.
+
+To integrate Panel effectively, ensure the installation of `jupyter_bokeh` by executing `pip install jupyter_bokeh` or `conda install -c bokeh jupyter_bokeh`. Subsequently, activate the extension using `pn.extension()`.
```

Comment:
Is it `notebook`, `jupyter_bokeh` or both that should be installed?

Reply:
For me Panel in notebooks in PyCharm only works if I install both "jupyter_bokeh" and "jupyter".

They recommend installing the "jupyter" package in the docs:
https://www.jetbrains.com/help/pycharm/ipython-notebook-support.html#get-started

And in the app as well:

<img width="677" alt="Screenshot 2024-03-29 at 10 30 12" src="https://github.com/holoviz/panel/assets/852409/d8ab8d1d-7ace-41d9-95df-9d344d51f65e">

Although for me just installing "notebook" it seems it works equally well.

Reply:
`notebook` is the older metapackage for the Jupyter server and notebook frontend. In recent versions of Jupyter notebooks that uses the JupyterLab based frontend they are effectively equivalent.